task8/src/main/java/PostWork/Service.java

```diff
@@ -0,0 +1,206 @@
+import java.util.Map;
+
+import com.google.gson.Gson;
+
+public class Service {
+    private List<Post> PList = new ArrayList<Post>(Arrays.asList(
+            new Post(1,
+                    "Bag number 1, vintage, Italian quality",
+                    "2021-03-17T23:00:00",
+                    "2021-10-17T23:00:00",
+                    "Paseko Nadia",
+                    "https://www.dior.com/en_int/products/couture-M0565OJAP_M885-medium-lady-d-lite-bag-yellow-multicolor-dior-paisley-embroidery",
+                    "30%",
+                    new ArrayList<String>(Arrays.asList("#bag", "#Dior")),
+
+            new Post(2,
+                    "Bag number 2, vintage, Italian quality",
+                    "2021-03-17T23:00:00",
+                    "2021-10-17T23:00:00",
+                    "Paseko Nadia",
+                    "https://www.dior.com/en_int/products/couture-M0565OJAP_M885-medium-lady-d-lite-bag-yellow-multicolor-dior-paisley-embroidery",
+                    "20%",
+                    new ArrayList<String>(Arrays.asList("#bag")),
+
+            new Post(3,
+                    "Bag number 3, vintage, Italian quality",
+                    "2021-03-17T23:00:00",
+                    "2021-10-17T23:00:00",
+                    "Paseko Nadia",
+                    "https://www.dior.com/en_int/products/couture-M0565OJAP_M885-medium-lady-d-lite-bag-yellow-multicolor-dior-paisley-embroidery",
+                    "10%",
+                    new ArrayList<String>(Arrays.asList("#bag", "#Dior")),
+
+            new Post(4,
+                    "Bag number 4, vintage, Italian quality",
+                    "2021-03-17T23:00:00",
+                    "2021-10-17T23:00:00",
+                    "Paseko Nadia",
+                    "https://www.dior.com/en_int/products/couture-M0565OJAP_M885-medium-lady-d-lite-bag-yellow-multicolor-dior-paisley-embroidery",
+                    "15%",
+                     new ArrayList<String>(Arrays.asList("#bag", "#Dior")),
+
+            new Post(5,
+                    "Bag number 5, vintage, Italian quality",
+                    "2021-03-17T23:00:00",
+                    "2021-10-17T23:00:00",
+                    "Paseko Nadia",
+                    "https://www.dior.com/en_int/products/couture-M0565OJAP_M885-medium-lady-d-lite-bag-yellow-multicolor-dior-paisley-embroidery",
+                    "10%",
+                    new ArrayList<String>(Arrays.asList("#bag", "#Dior")),
+
+            new Post(12,
+                    "Bag number 12, vintage, Italian quality",
+                    "2021-03-17T23:00:00",
+                    "2021-10-17T23:00:00",
+                    "Paseko Nadia",
+                    "https://www.dior.com/en_int/products/couture-M0565OJAP_M885-medium-lady-d-lite-bag-yellow-multicolor-dior-paisley-embroidery",
+                    "15%",
+                    new ArrayList<String>(Arrays.asList("#bag", "#Dior")));
+
+
+    public static Service INSTANCE = null;
```

Comment:
1) This field must be made private. Because there's a getter beside it. If we declare a getter, we are willing to hide the field itself from an outer consumer. Because we've just created the getter for that purpose)

2) Such kind of initialization of a static field is not thread-safe. Just make your field final and initialize it like `private static final Service INSTANCE = new Service();`
Surprisingly, less code gives a more reliable result in this case